cmd/agent/app/jmx.go

```diff
@@ -122,38 +122,28 @@ func doJmxListNotCollected(cmd *cobra.Command, args []string) error {
 	return runJmxCommand("list_not_matching_attributes")
 }
 
-func setupAgent() error {
-	overrides := make(map[string]interface{})
-
-	// let the os assign an available port
-	overrides["cmd_port"] = 0
+func runJmxCommand(command string) error {
 
+	overrides := make(map[string]interface{})
+	overrides["cmd_port"] = 0 // let the os assign an available port
 	config.SetOverrides(overrides)
+
 	err := common.SetupConfig(confFilePath)
 	if err != nil {
 		return fmt.Errorf("unable to set up global agent configuration: %v", err)
 	}
 
-	common.SetupAutoConfig(config.Datadog.GetString("confd_path"))
-
-	// start the cmd HTTP server
-	if err := api.StartServer(); err != nil {
-		return fmt.Errorf("Error while starting api server, exiting: %v", err)
-	}
-
-	return nil
-}
-
-func runJmxCommand(command string) error {
-	err := config.SetupLogger(loggerName, jmxLogLevel, "", "", false, true, false)
+	err = config.SetupLogger(loggerName, config.GetEnv("DD_LOG_LEVEL", "off"), "", "", false, true, false)
```

Comment:
I can re-add it, do you think it's worth keeping?

Reply:
The way I see it is: it was used to defined the logging level on this command and now it's not anymore so that kind of a breaking change without depreciation warning.

Reply:
Done.